src/actions/auth.actions.ts

```diff
@@ -53,3 +66,33 @@ export const loginHandler = (payload: ILoginPayload) => {
 		}
 	};
 };
+
+
+export interface IVerificationPayload {
+	verificationCode: Number;
+}
+
+/**
+ * This function will send request to `/auth/verify` route
+ * @param payload for post request
+ * @author is-it-ayush
+ */
+
+export const verifyHandler = (payload: IVerificationPayload, token: String) => {
+	return async () => {
+		interface APIResponse extends IAPIResponseSuccess {
+			message: string;
+		}
+
+		try {
+			const res = await API.get<APIResponse>(`/auth/verify/${payload.verificationCode}`, {
+				headers: {
+					"authorization": `Bearer ${token}`,
```

Comment:
Make `a` of Authorization upper case  `A`

Reply:
Sure!